src/queue.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+export default class Queue {
+  constructor(initialValues=[]) {
+    this.elements = initialValues
+    this.first = initialValues[0]
```

Comment:
Why are you keeping track of `this.first` as an instance variable? This is unnecessary since you're storing your Queue elements in an array.